charts/lotus-seed/templates/statefulset.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.enabled }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name }}-lotus-seed
+  labels:
+    app: lotus-seed-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ .Release.Name }}-lotus-seed-service
+  selector:
+    matchLabels:
+      app: lotus-seed-app
+  template:
+    metadata:
+      labels:
+        app: lotus-seed-app
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      hostNetwork: {{ .Values.hostNetwork }}
+{{- if .Values.securityContext.runAsNonRoot }}
+      securityContext:
+        fsGroup: 532
+        runAsNonRoot: true
+        runAsUser: 532
+        runAsGroup: 532
+{{- end }}
+      volumes:
+        - name: lotus-seed-config-volume
+          configMap:
+            name: {{ .Release.Name}}-seed-nginx-config
+            items:
+            - key: server.conf.template
+              path: server.conf.template
+      initContainers:
+{{- if .Values.generate.enabled }}
+{{- range .Values.minerIDs }}
+      - name: generate-{{ . }}
+        image: {{ $.Values.image.lotusSeed.repository }}:{{ $.Values.image.lotusSeed.tag }}
+        imagePullPolicy: {{ $.Values.image.lotusSeed.pullPolicy }}
+        command: ["/bin/sh","-c"]
+        args:
+          - |
+            if [ -d "/sectors/{{ . }}" ] && [ -z "{{ $.Values.recreateSectors }}" ]; then
+              echo "skipping seed sector generation"
+            else
+              mkdir -p /sectors/{{ . }}
+              rm -rf /sectors/{{ . }}/*
+              lotus-seed --sector-dir=/sectors/{{ . }} pre-seal --miner-addr={{ . }} --sector-size 2KiB --num-sectors=32 --fake-sectors
+
+              cd /sectors/{{ . }}
+              tar -cvzf pre-seal-sectors-{{ . }}.tar.gz sealed
```

Comment:
We basically need everything inside of `/sectors/{{ . }}`. The `pre-seal-*.json` and `pre-seal-*.key` can be ignored but we need the `sealed`, `cache`, `unsealed` (nothing in it but I think we might need the directory), and the `sectorstore.json` file.